client/gutenberg/extensions/related-posts/index.js

```diff
@@ -0,0 +1,77 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { includes } from 'lodash';
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
```

Comment:
Suppose this contains styles meant for both theme side & the editor?

Since this block doesn't output anything yet, it might not make sense to do this just now, but in general:

- We can add `view.js` and import `view.scss` from there. As a result, we'll get `view.css`
- We don't need to import this style file from `editor.js` since the view.css will be loaded in the editor anyway.

Reply:
Yup, that's a good point. And I'll eventually go that way as I iterate on the block to make related posts be dynamically pulled in the editor.